frontend/package.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Update the workflow to diital ocean and add the one that pushes code to heroku for deployment on backend merge to develop branch and on main branch as well